addons/resource.language.en_gb/resources/strings.po

```diff
@@ -9491,7 +9491,15 @@ msgctxt "#19241"
 msgid "The PVR manager has been enabled without any enabled PVR add-on. Enable at least one add-on in order to use the PVR functionality."
 msgstr ""
 
-#empty strings from id 19242 to 19243
+#: xbmc/pvr/timers/PVRTimerInfoTag.cpp
+msgctxt "#19242"
+msgid "%d will record"
```

Comment:
@Jalle19 @xhaggi - Problems with 'upcoming'
1) 1 upcoming  / 2,3,4,etc... upcomings - i.e. two different strings needed. Not necessarily a problem but...
2) You can set pvr.mythtv to include 'inactive' matches for each rule (repeats, duplicates etc...) in the timers list. This PR logic currently excludes child timers with 'won't record' states (conflict_nok etc..).  I think 'will record' more clearly explains why these 'inactive' child timers aren't included in the count.
I'm open to further discussion / suggestion but still think with the current PRs logic 'will record' is better.